carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/utils/android/AndroidService.java

```diff
@@ -415,7 +416,7 @@ public boolean findExpectedNotification(String expectedTitle, String expectedTex
     public boolean findExpectedNotification(String expectedTitle, String expectedText, boolean partially) {
         // open notification
         try {
-            ((AndroidDriver) castDriver()).openNotifications();
+            DriverListener.castDriver(getDriver(), AndroidDriver.class).openNotifications();
```

Comment:
DriverListener from carina-webdriver shouldn't be imported in carina-utils!

Reply:
done